.github/workflows/ci.yml

```diff
@@ -53,7 +48,18 @@ jobs:
 
       - name: "Get aptly version"
         run: |
-           make version
+           make version > VERSION
```

Comment:
the `VERSION` file is generated in debina/rules and the Makefile, the pipeline should avoid writing to the git repo directly I think...